server/main.py

```diff
@@ -17,6 +22,19 @@ async def register(websocket):
 
 
 async def handler(websocket):
+    init = RequestEvent(type="init", data="Provide a username")
```

Comment:
I think all of this could be made into an `initialize_player` function and called before the connection is registered in `register`. This would ensure that the client doesn't receive a broadcast message by mistake before it gets initialized.

Reply:
Sounds good